packages/icons/src/iconVariants.ts

```diff
@@ -0,0 +1,70 @@
+    action: require('../assets/icons/action.svg'),
+    backspace: require('../assets/icons/backspace.svg'),
+    biometric: require('../assets/icons/biometric.svg'),
+    calendar: require('../assets/icons/calendar.svg'),
+    check: require('../assets/icons/check.svg'),
+    chevronDown: require('../assets/icons/chevronDown.svg'),
+    chevronLeft: require('../assets/icons/chevronLeft.svg'),
+    chevronRight: require('../assets/icons/chevronRight.svg'),
+    close: require('../assets/icons/close.svg'),
+    closeCircle: require('../assets/icons/closeCircle.svg'),
+    eyeglasses: require('../assets/icons/eyeglasses.svg'),
+    flag: require('../assets/icons/flag.svg'),
+    ghost: require('../assets/icons/ghost.svg'),
+    hiddenWallet: require('../assets/icons/hiddenWallet.svg'),
+    home: require('../assets/icons/home.svg'),
+    label: require('../assets/icons/label.svg'),
+    lock: require('../assets/icons/lock.svg'),
+    notifications: require('../assets/icons/notifications.svg'),
+    palette: require('../assets/icons/palette.svg'),
+    paper: require('../assets/icons/paper.svg'),
+    placeholder: require('../assets/icons/placeholder.svg'),
+    plus: require('../assets/icons/plus.svg'),
+    prices: require('../assets/icons/prices.svg'),
+    qrCode: require('../assets/icons/qrCode.svg'),
+    question: require('../assets/icons/question.svg'),
+    receive: require('../assets/icons/receive.svg'),
+    reject: require('../assets/icons/reject.svg'),
+    search: require('../assets/icons/search.svg'),
+    send: require('../assets/icons/send.svg'),
+    settings: require('../assets/icons/settings.svg'),
+    standardWallet: require('../assets/icons/standardWallet.svg'),
+    tor: require('../assets/icons/tor.svg'),
+    trade: require('../assets/icons/trade.svg'),
+    warningCircle: require('../assets/icons/warningCircle.svg'),
+} as const;
+
+export const cryptoIcons = {
+    bch: require('../assets/cryptoIcons/bch.svg'),
+    btc: require('../assets/cryptoIcons/btc.svg'),
+    btg: require('../assets/cryptoIcons/btg.svg'),
+    dash: require('../assets/cryptoIcons/dash.svg'),
+    dgb: require('../assets/cryptoIcons/dgb.svg'),
+    doge: require('../assets/cryptoIcons/doge.svg'),
+    etc: require('../assets/cryptoIcons/etc.svg'),
+    eth: require('../assets/cryptoIcons/eth.svg'),
+    gnt: require('../assets/cryptoIcons/gnt.svg'),
+    ltc: require('../assets/cryptoIcons/ltc.svg'),
+    name: require('../assets/cryptoIcons/name.svg'),
+    test: require('../assets/cryptoIcons/test.svg'),
+    trop: require('../assets/cryptoIcons/trop.svg'),
+    txrp: require('../assets/cryptoIcons/txrp.svg'),
+    vtc: require('../assets/cryptoIcons/vtc.svg'),
+    xrp: require('../assets/cryptoIcons/xrp.svg'),
+    zec: require('../assets/cryptoIcons/zec.svg'),
+} as const;
+
+export const flagIcons = {
+    cz: require('../assets/flags/cz.svg'),
+} as const;
+
+export type Icons = keyof typeof icons;
```

Comment:
Please use same name as prop `name` so type will be `IconName`. Current name is confusing because it differers from component name by just 's'.

Also this 's' plular should be singular because union type represents only one value at given time. `const name: IconName = 'close'`

This applies to all types here inc. Crypto, FlagIcon...